test/functional/read_directory.js

```diff
@@ -1,7 +1,7 @@
 const Category = require('../../lib/category');
 const MarkdownRenderer = require('../../lib/md-renderer');
 const path = require('path');
-const createDummyConfig = require('../unit/zendesk-uploader/test-utils').createDummyConfig;
+const createDummyConfig = require('../unit/test-utils').createDummyConfig;
```

Comment:
Anyway, this thing `../unit/test-utils../unit/test-utils` looks weird. 
I actually not sure what `../unit/test-utils` appendix means and how is it being processed without any errors?